contracts/governance/Governor.sol

```diff
@@ -21,8 +21,8 @@ import {IGovernor, IERC6372} from "./IGovernor.sol";
  *
  * This contract is abstract and requires several functions to be implemented in various modules:
  *
- * - A counting module must implement {quorum}, {_quorumReached}, {_voteSucceeded} and {_countVote}
- * - A voting module must implement {_getVotes}
+ * - A counting module must implement {proposalVotes}, {_quorumReached}, {_voteSucceeded} and {_countVote}
+ * - A voting module must implement {_getVotes} and {quorum}
```

Comment:
{quorum} should not be described here IMO. Its like `votingDelay` or `votingPeriod`. The IGovernor documentation marks it as `@notice module:user-config`
```suggestion
 * - A voting module must implement {_getVotes}
```

Reply:
Updated. I've added the rest of the virtual function in the third bullet point but I was also considering to remove that bullet point altogether as it can easily go out of sync.